### PR TITLE
ux: Small tweaks ✨ 

### PR DESCRIPTION
**Pull request by NunoAlexandre**

### CODE REVIEW

File: app/src/components/TokenSpendApproval.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const TokenSpendApproval: FC<Props> = ({ onClick, approving }) => {
     <div
       className={cn(
         'mt-8 rounded-[20px] bg-turtle-secondary-light px-3 py-5',
-        approving ? 'animate-pulse' : '',
+        approving ? 'animate-pulse cursor-progress' : '',
```
Comment (noahjoeris) [inline app/src/components/TokenSpendApproval.tsx:16]: When I have a loading cursor my first reaction is always the app got stuck.

File: app/src/components/TokenSpendApproval.tsx [hunk 2]
```diff
@@ -25,7 +25,10 @@ const TokenSpendApproval: FC<Props> = ({ onClick, approving }) => {
           <div className="pb-2 text-left text-small text-turtle-foreground">
             We first need your approval to transfer this token from your wallet.
             <button
-              className="ml-1 text-left text-small text-turtle-foreground underline"
+              className={cn(
+                'ml-1 text-left text-small text-turtle-foreground underline',
+                approving ? 'cursor-not-allowed' : '',
```
Comment (noahjoeris) [inline app/src/components/TokenSpendApproval.tsx:30]: Do you like `not allowed cursors`? 😅 Personally I find them very ugly.

**Review by noahjoeris — APPROVED** [review]
Good eyes Nuno 👁️✔️

File: app/src/components/TokenSpendApproval.tsx (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/components/TokenSpendApproval.tsx:16]: That's unusual I'd say. I would think that if the component is disabled while the cursor is still displaying normally when hoovering it.

File: app/src/components/TokenSpendApproval.tsx (hunk 2 above)
Comment (NunoAlexandre) [reply app/src/components/TokenSpendApproval.tsx:30]: I think it's fine, don't find them ugly per se. But we can just use `cursor-progress` here too. Do you prefer that?

File: app/src/components/TokenSpendApproval.tsx (hunk 1 above)
Comment (noahjoeris) [reply app/src/components/TokenSpendApproval.tsx:16]: I am unusual haha 😄 Let's go then

File: app/src/components/TokenSpendApproval.tsx (hunk 2 above)
Comment (noahjoeris) [reply app/src/components/TokenSpendApproval.tsx:30]: nah then keep the `not allowed cursor`

File: app/src/components/TokenSpendApproval.tsx (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/components/TokenSpendApproval.tsx:16]: You are special 💘

Comment (Victor-Salomon) [reply app/src/components/TokenSpendApproval.tsx:16]: I do not feel cursors wrong. 
At least it prevents from crazy moves.

**Review by Victor-Salomon — APPROVED** [review]
Lets gooo

**Pull request merged**